lexoid/core/utils.py

```diff
@@ -45,6 +45,17 @@ def split_pdf(input_path: str, output_dir: str, pages_per_split: int):
     return paths
 
 
+def create_sub_pdf(
+    input_path: str, output_path: str, page_nums: Optional[tuple] = None
+) -> str:
+    with pikepdf.open(input_path) as pdf:
+        indices = page_nums if page_nums else range(len(pdf.pages))
```

Comment:
How do we feel about updating line:52 as?
```
  if isinstance(page_nums, int):
      page_nums = [page_nums]
  indices = list(set(page_nums)) if page_nums else range(len(pdf.pages))
```

Accordingly, tests could be updated
```
    result = parse(sample, "STATIC_PARSE", page_nums=(2), pages_per_split=1)
    assert len(result["segments"]) == 1
    assert "ATTEMPTS" in result["raw"]
    assert "acp@dca.ca.gov" not in result["raw"]

    result = parse(sample, "STATIC_PARSE", page_nums=2, pages_per_split=1)
    assert len(result["segments"]) == 1
    assert "ATTEMPTS" in result["raw"]
    assert "acp@dca.ca.gov" not in result["raw"]
```

Reply:
Added

Reply:
Duplicate scenario (1, 1) will cause parsing twice?

Reply:
Fixed